packages/blade/src/components/Collapsible/Collapsible.stories.internal.tsx

```diff
@@ -29,14 +29,19 @@ const Page = (): ReactElement => {
       <Title>Usage</Title>
       <Sandbox editorHeight={500}>
         {`
-        import { Collapsible, CollapsibleButton, CollapsibleBody, Text } from '@razorpay/blade/components';
+        import { Collapsible, CollapsibleButton, CollapsibleBody, Text, Amount } from '@razorpay/blade/components';
 
         function App() {
           return (
             <Collapsible>
-              <CollapsibleButton>Answer to life, universe and everything</CollapsibleButton>
+              <CollapsibleButton>View Price Breakdown</CollapsibleButton>
```

Comment:
probably need to do some flex thing to make it look good?

<img width="336" alt="Screenshot 2023-06-16 at 10 54 17 AM" src="https://github.com/razorpay/blade/assets/11384858/9585fe08-2ee6-4a04-8382-adec0a6fa97b">


also, ask design to give you better version of price breakup we have one use case from dashboard. @saurav12 can you give Isha's use case?

Reply:
Have requested on Figma, will update the storybook based on that.

Reply:
sure but atleast align the contents right now it looks visually broken

probably can use this example - https://razorpay.enterprise.slack.com/files/UJPA0H3K9/F05CX5B8EDS/image.png

Reply:
Updated in https://github.com/razorpay/blade/pull/1296/commits/ded03e96d4dbf444265b3a0d2d0b22bd8302491d

![image](https://github.com/razorpay/blade/assets/6682655/1bc450c9-d428-4fb8-b615-89d41dc36537)